test/test_sparse.py

```diff
@@ -1200,6 +1200,19 @@ def _test_basic_ops_shape(self, nnz_x1, nnz_x2, shape_i, shape_v=None):
         self.assertEqual(self.safeToDense(y1), expected)
         self.assertEqual(self.safeToDense(y2), expected)
 
+        # Note: true_divide does not have a method variant
+        y1 = torch.true_divide(x1, 37.5)
```

Comment:
wait, why did we add a method for floor_divide but not true_divide?

Reply:
Both true_divide and floor_divide are only functions, not methods, in NumPy. I think you actually brought this up when floor_divide was first being implemented. When true_divide was implemented it respected the same convention.

When we look forward to deprecating integer division using torch.div, however, we don't want users to lose functionality. In particular, we want to provide a mechanism to perform in-place integer division, hence Tensor.floor_divide_. 

We could add Tensor.true_divide for consistency. We decided at the time that it was mostly moot since in the near future Tensor.div would be Tensor.true_divide, anyway.

Reply:
In the "future_div" PR I'll make true_divide a method.

Reply:
See https://github.com/pytorch/pytorch/pull/34794